src/components/TextArea/index.less

```diff
@@ -57,12 +57,10 @@
       height: 100%;
       width: 100%;
       resize: none;
-      color: inherit;
       overflow: hidden;
       outline: 0;
       padding: 10px;
       -webkit-font-smoothing: antialiased;
-      -webkit-text-fill-color: transparent;
```

Comment:
This cannot be deleted, and the highlighting of the editing area is invalid.

Reply:
The overlay as it current is implemented does not work for all content. Try my example with spans in the content using an editable div and you'll see that the transparent text of the div does not align properly with the visible text of the pre. I have therefore removed the entire overlay, and make the div/textarea text non-transparent.

The user can instead do syntax highlighting in the markdown preview pane by adding their own highlighting pass to the plugins.

Reply:
Your idea will invalidate the editor code highlighting.

Reply:
correct highlighting in the textarea/div side is removed. Is this not acceptable?

I was unable to fix the overlay alignment with css, perhaps you could try? If it is fixable it won't need to be removed. Just use a contenteditable div and then out spans inside it

Reply:
Add a new API, to be forward compatible.